### PR TITLE
Added auto webhook support

### PR DESCRIPTION
**Pull request by ChetanGN**

### CODE REVIEW

File: razorpay/razorpay.php [hunk 1]
```diff
@@ -1,44 +1,47 @@
 <?php
 
 require_once __DIR__.'/razorpay-sdk/Razorpay.php';
+require_once __DIR__.'/razorpay-webhook.php';
```
Comment (ramth05) [inline razorpay/razorpay.php:4]: why we need this

Comment (ChetanGN) [reply razorpay/razorpay.php:4]: @ramth05 instead of hardcoding the event name, considering it from the const defined in razorpay-webhook.php file.

Comment (ramth05) [reply razorpay/razorpay.php:4]: ok, let define the events in this file only. As we already including this file in webhook script.

**Review by ramth05 — APPROVED** [review]
LGTM, Please test it properly.

File: razorpay/razorpay.php [hunk 2]
```diff
@@ -448,26 +390,122 @@ private function _postProcess()
...
+
+        if(empty($eventsSubscribe) == false)
+        {
+            foreach ($eventsSubscribe as $value) 
+            {
+                $prepareEventsData[$value] = true;
+            }
+        }
+
+        if(in_array($_SERVER['SERVER_ADDR'], ["127.0.0.1","::1"]))
```
Comment (ramth05) [inline razorpay/razorpay.php:420]: This will fail in case of server is running on localhost with IP published.

**Review by ramth05 — CHANGES_REQUESTED** [review]
Please apply local host change.

**Review by jampulanaveen — APPROVED** [review]

**Pull request merged**